### ISSUE TITLE
Scrollbar Not Available on Code/Mem Views & Sybol Search Does Not Honor Case Sensitivity

### ISSUE THREAD

**Issue by paulhocker** [Sakrac/IceBroLite#32]
It would be nice to have the scrollbar functioning in the code and memory view. Right now they seem to not be available or appear briefly and then dissapear when using the mouse scroll wheel.

https://user-images.githubusercontent.com/2214652/210140246-701a8cc3-83ba-4afc-9984-2f6d4594899a.mp4

The symbols view does not seem to take into account the Case insensitivity switch (I assume thats what that is)

https://user-images.githubusercontent.com/2214652/210140348-c7b2e08f-9f58-4c89-be52-8633ccbe4722.mp4

**Comment by Sakrac** [comment]
missed a place for case sensitivity in the wildcard search so that was easy
scrollbar for code is harder though because it doesn't have a start and end, maybe I can just pick some range, will think about it.

**Comment by Sakrac** [comment]
A custom vertical scrollbar would conflict with the horizontal scrollbar, so instead allowing middle mouse button dragging on code and memory window contents. Not quite as intuitive but hoping it is a suitable solution.

**Comment by paulhocker** [comment]
It is much better than using the scroll wheel, but still does not feel quite right.

Why do the code/mem views not have a start and end range or 0-65536?

Or is it just showing a small portion of memory at a time?

**Comment by Sakrac** [comment]
There are two issues, one is that the horizontal and vertical scrollbar are tied together, but the larger issue is that if the window shows 512 bytes the scrollbar would be 1/128 of the window height so it would be hard to grab and moving it one pixel could move the entire screen making it hard to follow the scrolling. But maybe I'll add a custom warping scroll bar
For the code view I also can't tell how many lines of machine code the entire memory represents.
